#! /bin/sh -e
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changes confusing reference to --md5 and --sha. Bug#274146 

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0
@DPATCH@

diff -urN prelink-0.0.20040908.orig/doc/prelink.8 prelink-0.0.20040908/doc/prelink.8
--- prelink-0.0.20040908.orig/doc/prelink.8	2003-08-05 08:27:59.000000000 +0000
+++ prelink-0.0.20040908/doc/prelink.8	2004-10-01 14:43:24.000000000 +0000
@@ -173,7 +173,7 @@
 unprobable case of modified file and original file having the same digest
 or checksum).
 .TP
-.B \-\-md5
+.B \-y \-\-md5
 This is similar to
 .I \-\-verify
 option, except instead of outputing the content of the binary or library
@@ -181,7 +181,7 @@
 See
 .BR md5sum (1).
 .TP
-.B \-\-sha
+.B \-y \-\-sha
 This is similar to
 .I \-\-verify
 option, except instead of outputing the content of the binary or library
